spec/core/collection_spec.rb

```diff
@@ -49,6 +49,12 @@
       assert_requested(:put, %r{bulk_test_resources/update_many\?ids=1,2,3$})
     end
 
+    it "should defer #create_or_update to the resource class" do
+      resource = ZendeskAPI::Collection.new(client, ZendeskAPI::CreateOrUpdateTestResource)
+      stub_json_request(:post, %r{create_or_update_test_resources/create_or_update}, json(create_or_update_test_resource: { param: "abc" }))
+      resource.create_or_update!
```

Comment:
@sergiy-koshoviy is this tests finished? shouldn't we assert that the request is taking the expected parameters here?

Reply:
I think so, I will address myself.